src/main/java/spring/controller/OrderController.java

```diff
@@ -37,14 +31,4 @@ public List<Order> getAllOrder() {
     public Order addOrder(@RequestBody Order order) {
         return orderService.addOrder(order);
     }
-
-    @PostMapping("/product")
-    public Product addProduct(@RequestBody Product product) {
-        return productService.addProduct(product);
-    }
-
-    @PostMapping("/product/{id}")
```

Comment:
if you want to get a product from particular order  then the path will be like this "{id}/product/{productId}"